src/components/TokenBalance.tsx

```diff
@@ -121,14 +121,18 @@ function TokenBalance({
   }
 }
 
-function HideBalanceButton({ hideBalance }: { hideBalance: boolean }) {
+export function HideBalanceButton({ hideBalance }: { hideBalance: boolean }) {
   const dispatch = useDispatch()
   const eyeIconOnPress = () => {
     AppAnalytics.track(hideBalance ? HomeEvents.show_balances : HomeEvents.hide_balances)
     dispatch(toggleHideBalances())
   }
   return (
-    <Touchable onPress={eyeIconOnPress} hitSlop={variables.iconHitslop}>
+    <Touchable
+      onPress={eyeIconOnPress}
+      hitSlop={variables.iconHitslop}
+      testID={'HideBalanceButton'}
+    >
```

Comment:
nit: Adding a high border-radius here will make the touchable ripple a circle and looks more like an eye.

```suggestion
    borderRadius={30}
    >
```

| Before | After |
| ----- | ----- |
| ![](https://github.com/user-attachments/assets/09bd9118-0381-4fbf-96a6-70557b58127b) | ![](https://github.com/user-attachments/assets/c9207e2a-f674-4cc2-807c-24b4c5f808bb) |